frontend/tailwind.config.js

```diff
@@ -35,13 +35,9 @@ module.exports = {
         hopeone: "url('../public/images/bg/page_3_bg.png')",
         hopetwo: "url('../public/images/bg/page_4_7_bg.png') ",
         hopethree: "url('../public/images/bg/page_8_9_bg.png') ",
-<<<<<<< HEAD
         hopefour: "url('src/assets/images/page_11_31_bg.png')",
-      },
-=======
         'bg-name': "url('/assets/images/bg-log-in.png')",
```

Comment:
merge this #104 and use the pre-set, it should help

Reply:
merged